spec/features/javascript/block_controls_spec.rb

```diff
@@ -25,6 +25,12 @@
     # click to add widget
     click_add_widget
 
+    # Check if the Sir Trevor icons are loading. They are in the same SVG so a single check should be sufficient.
+    within('.st-block-replacer') do
+      href_value = find('use')['xlink:href']
+      expect(href_value).not_to eq('undefined#add-block')
```

Comment:
Would it be better to check for the presence of `*.svg#add-block`?

Reply:
Yes, that's better.